.github/workflows/test-users.yml

```diff
@@ -0,0 +1,332 @@
+# Test a selection of projects which depend on rustix.
+
+name: Test rustix's users
+
+on: workflow_dispatch
+
+jobs:
+  async-io:
+    name: async-io ported to rustix
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        build: [stable]
+        include:
+          - build: stable
+            os: ubuntu-latest
+            rust: stable
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - uses: ./.github/actions/install-rust
+      with:
+        toolchain: ${{ matrix.rust }}
+
+    - run: sed -i 's/^version = "\([[:digit:].]*\)-alpha.0"$/version = "\1"/' Cargo.toml
+    - run: git clone https://github.com/sunfishcode/async-io
```

Comment:
I tend to use `--depth=1` for these types of things; doesn't matter with this repo right now, but does start to matter with much longer lived ones.  But even higher level I think it makes sense to use the `checkout` action as is [done here](https://github.com/containers/containers-image-proxy-rs/blob/3d83477f527fc8791da699ddce5f2605aeb20beb/.github/workflows/ci.yaml#L54) which in theory I think can e.g. do caching behind the scenes (and might do so).

Reply:
Good idea, using the `checkout` action seems straightforward, so I've now changed the script to use that.